openpype/hosts/fusion/plugins/load/load_fbx.py

```diff
@@ -15,7 +14,21 @@ class FusionLoadFBXMesh(load.LoaderPlugin):
 
     families = ["*"]
     representations = ["*"]
-    extensions = {"fbx"}
+    extensions = {
+        "3ds",
+        "amc",
+        "aoa",
+        "asf",
+        "bvh",
+        "c3d",
+        "dae",
+        "dxf",
+        "fbx",
+        "htr",
+        "mcd",
+        "obj",
+        "trc",
+    }
```

Comment:
Anyone else feels its weird that a `FusionLoadFBXMesh` loader now loads all these extensions? Does the `SurfaceFBXMesh` tool support all these extensions? 😮